docs/SUMMARY.md

```diff
@@ -23,7 +23,7 @@
 * [Running Ambassador](user-guide/running.md)
 * [Upgrading Ambassador](user-guide/upgrading.md)
 * [Configuring Ambassador](reference/configuration.md)
-* [Auth with an external auth service](how-to/auth-external.md)
+* [Auth with an External Auth Service](how-to/auth-external.md)
```

Comment:
Thanks very much for tackling this! Don't let my playing style editor here detract from my appreciation for doing the work. :)

I think capitalizing "With" in titles ends up making sense for us, so it should be capitalized here.